sky/cli.py

```diff
@@ -4359,6 +4364,99 @@ def _delete_benchmark(benchmark: str) -> None:
         progress.refresh()
 
 
+@cli.group(cls=_NaturalOrderGroup, hidden=True)
+def local():
+    """SkyPilot local tools CLI."""
+    pass
+
+
+@local.command('up', cls=_DocumentedCodeCommand)
+@usage_lib.entrypoint
+def local_up():
```

Comment:
Yeah, that's a good point. I want to avoid using `launch` because  I want users to associate `launch` with SkyPilot clusters. This CLI should use similar, but not the same, commands, hence I choose `up`.  wdyt?